contrib/internal/sqltest/sqltest.go

```diff
@@ -125,9 +128,10 @@ func testQuery(cfg *Config) func(*testing.T) {
 
 		spans := cfg.mockTracer.FinishedSpans()
 		var querySpan mocktracer.Span
+		expectedComment := "/*dde='test-env',ddsid='test-span-id',ddsn='test-service',ddsp='0',ddsv='v-test',ddtid='test-trace-id'*/"
 		if cfg.DriverName == "sqlserver" {
 			//The mssql driver doesn't support non-prepared queries so there are 3 spans
-			//connect, prepare, and query
+			//connect, prepare and query
```

Comment:
nit: why remove the oxford comma here? Not a fan of them? :P

Reply:
I don't actually remember doing this. I'll revert.